app/core/src/main/java/com/fsck/k9/mailstore/MessageViewInfo.java

```diff
@@ -26,7 +26,7 @@
     public MessageViewInfo(
             Message message, boolean isMessageIncomplete, Part rootPart,
             String subject, boolean isSubjectEncrypted,
-            String text, String plainText, List<AttachmentViewInfo> attachments,
+            String text, String textPlainFormatted, List<AttachmentViewInfo> attachments,
```

Comment:
``` textPlainFormatted``` can be declared without re-writing the variable type.
```String text, textPlainFormatted```

Reply:
Thank you for your suggestion, I will have a look into it in the next days.

Reply:
To be honest, I do not know the concept of defining a list of parameters with the same type without repeating the type identifier. Especially when the parameter list is not a the end of the methods parameter list. As I am eager to explore new knowledge I gave it a try. It results in an error:
```
k-9/app/core/src/main/java/com/fsck/k9/mailstore/MessageViewInfo.java:29: error: <identifier> expected
            String text, textPlainFormatted, List<AttachmentViewInfo> attachments,
                                           ^
1 error
```

Could you please point me to the specific part of the language reference where this is explained?

Reply:
There is no precedence in the current code base for such an idiom and the [code style](https://github.com/k9mail/k-9/wiki/CodeStyle#java) does not mention it. I do not see a very good reason for this change. In order not to break the code style I will not apply the suggested change and mark this conversation as resolved.

I will reconsider the [suggested change](https://github.com/k9mail/k-9/pull/3936/files/21e002d6b8ed6577b23d1f2e183281d8efa4f170#r265710231) in case there is a statement by a [K-9 Mail team member](https://github.com/orgs/k9mail/people) or collaborator of this repository that suggests it. Please point me to such if known.